app/views/hub/clients/edit_13614c_form_page2.html.erb

```diff
@@ -248,3 +249,27 @@ scope of service chart)", yes_no_options_for_select) %>
     <% end %>
   </div>
 <% end %>
+<% content_for :script do %>
+  <<script>
+    document.addEventListener("DOMContentLoaded", function() {
+        let textArea = document.getElementById("other_income_additional_comments");
+        let prefix = "Other money received during the year includes: ";
+
+        if (!textArea.value.startsWith(prefix)) {
+            textArea.value = prefix + (textArea.value || '');
+        }
+
+        textArea.addEventListener("input", function() {
+            if (!this.value.startsWith(prefix)) {
+                this.value = prefix + this.value.replace(prefix, '');
+            }
+        });
+
+        textArea.form.addEventListener("submit", function() {
+            if (textArea.value.startsWith(prefix)) {
+                textArea.value = textArea.value.replace(prefix, '').trim();
```

Comment:
can `textArea.value` be `nil`? Or is it guaranteed to be `""`?

Reply:
I think its guaranteed to be at least "", I'm not sure how to check though